index.html

```diff
@@ -1,28 +1,34 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>My Portfolio</title>
-  <link rel="stylesheet" href="style.css">
-</head>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
 <body>
-  <nav class="navbar">
-    <h1>MY Logo</h1>
-    <ul class="nav-menu">
-      <li><a href="#">Home</a></li>
-      <li><a href="#">About</a></li>
-      <li><a href="#">Contact</a></li>
-    </ul>
-    <img  class="hamburger" src="assets/menu.png" width="50px"  alt="Logo" />
+  <header class="header">
+    <nav class="navbar">
+      <h1 ><a class="my-logo" href="#section2">MY Logo </a></h1>
+      <ul class="nav-menu">
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+      <img  class="hamburger" src="assets/menu.png" width="50px"  alt="Logo" />
     </nav>
-    <section class="section2-content">
-      <h1>I’m Melkamu Microverse student, Glad to see you!</h1>
-      <p>I’m a software developer! I can help you build a product feature or website Look through some of my 
+  </header>
```

Comment:
- [ ] [3]  To ensure compliance with the Figma design, Please promptly remove the header background from the project. I understand the importance of adhering to the provided design guidelines, and please make the necessary adjustments to ensure strict adherence to the Figma design. 😉